### PR TITLE
Fixing Reserved Balance

### PR DESCRIPTION
**Pull request by maayarosama**
### Description

Fixed reserved balance
### Changes

replaced reserved value with feeFrozen value

### Related Issues

#416

### CODE REVIEW

File: src/portal/components/FundsCard.vue [hunk 1]
```diff
@@ -63,26 +45,21 @@ export default class FundsCard extends Vue {
...
   @Prop({ required: true }) balanceReserved!: number;
   openBalance = false;
-  mounted() { 
-    this.$root.$on('updateBalance', (balance: number) => { 
-     this.balanceFree = balance;
-    })
+  mounted() {
+    this.$root.$on("updateBalance", (balance: number) => {
+      this.balanceFree = balance;
+      this;
```
Comment (rawdaGastan) [inline src/portal/components/FundsCard.vue]: should we remove this; ?

**Review by rawdaGastan — COMMENTED** [review]
github actions fail

**Review by rawdaGastan — COMMENTED** [review]
should I approve with failed github actions?

**Comment by maayarosama** [comment]
> should I approve with failed github actions?

I think so as not all files are linted

**Review by rawdaGastan — APPROVED** [review]

**Pull request merged**